src/theme.js

```diff
@@ -86,7 +89,7 @@ const reds = tints.map(tint(red))
 const oranges = tints.map(tint(orange))
 const purples = tints.map(tint(purple))
 
-export const colors = Object.assign({}, {
+export const colors = {...{
```

Comment:
I don't think this needs wrapping since it's not cloning another object

Reply:
updated